src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/appsettings.json

```diff
@@ -24,17 +24,24 @@
             "PasswordRequireUppercase": "false",
             "PasswordRequireLowercase": "false",
             "RequireUniqueEmail": "true",
-            "ConfirmationEmailResendDelay": "0.00:02:00", //Format: D.HH:mm:nn
-            "ResetPasswordEmailResendDelay": "0.00:02:00" //Format: D.HH:mm:nn
+            "ConfirmationEmailResendDelay": "0.00:02:00",
+            //Format: D.HH:mm:nn
+            "ResetPasswordEmailResendDelay": "0.00:02:00"
+            //Format: D.HH:mm:nn
         },
         "EmailSettings": {
-            "Host": "LocalFolder", // Local folder means storing emails in bin\sent-emails folder. (Recommended for testing purposes only)
+            "Host": "LocalFolder",
+            // Local folder means storing emails in bin\sent-emails folder. (Recommended for testing purposes only)
```

Comment:
again, please roll back all of the code formatting changes above this line, because these changes are not related to this PR.